assemblies/karaf-features/src/main/feature/feature.xml

```diff
@@ -186,6 +186,7 @@
     <bundle start-level="82">mvn:org.opencastproject/opencast-email-template-service-api/${project.version}</bundle>
     <bundle start-level="82">mvn:org.opencastproject/opencast-email-template-service-impl/${project.version}</bundle>
     <bundle start-level="82">mvn:org.opencastproject/opencast-engage-paella-player/${project.version}</bundle>
+    <bundle start-level="82">mvn:org.opencastproject/opencast-engage-serve-ui-config/${project.version}</bundle>
```

Comment:
This needs not only to be in allinone but in all of Opencast's distributions since they all contain some kind of user interface.

Reply:
moved to opencast-core